arm-mobileengagement/2014-12-01/swagger/mobile-engagement.json

```diff
@@ -1289,6 +1289,548 @@
+                        "description": "Control paging of export jobs, start results at the given offset, defaults to 0 (1st page of data).",
+                        "default": 0,
+                        "minimum": 0
+                    },
+                    {
+                        "name": "$top",
+                        "in": "query",
+                        "required": false,
+                        "type": "integer",
+                        "description": "Control paging of export jobs, number of export jobs to return with each call. By default, it returns all export jobs with a default paging of 20.\nThe response contains a `nextLink` property describing the path to get the next page if there are more results.\nThe maximum paging limit for $top is 40.",
+                        "default": 20,
+                        "maximum": 40,
+                        "minimum": 1
+                    },
+                    {
+                        "name": "$filter",
+                        "in": "query",
+                        "required": false,
+                        "type": "string",
+                        "description": "Filter can be used to restrict the results to export jobs matching a specific state.\nThe syntax is `$filter=state eq 'queued'`.\nValid state values are: queued, started, succeeded, and failed.\nOnly the eq operator and the state property are supported."
+                    },
+                    {
+                        "name": "$orderby",
+                        "in": "query",
+                        "required": false,
+                        "type": "string",
+                        "description": "Sort results by an expression which looks like `$orderby=jobId asc` (default when not specified).\nThe syntax is orderby={property} {direction} or just orderby={property}.\nProperties that can be specified for sorting: jobId, errorDetails, dateCreated, jobStatus, and dateCreated.\nThe available directions are asc (for ascending order) and desc (for descending order).\nWhen not specified the asc direction is used.\nOnly one orderby property can be specified."
+                    }
+                ],
+                "responses": {
+                    "200": {
+                        "description": "Success, export jobs are returned as JSON.",
+                        "schema": {
+                            "$ref": "#/definitions/ExportJobResultList"
+                        }
+                    },
+                    "default": {
+                        "description": "* `400`: Invalid parameters or application is disabled. Check response body for details.\n* `401`: Authentication error.",
+                        "schema": {
+                            "$ref": "#/definitions/ApiError"
+                        }
+                    }
+                },
+                "x-ms-pageable": {
+                    "nextLinkName": "nextLink"
+                }
+            },
+            "post": {
```

Comment:
This sounds like a long running operation. If it is, please LRO extensions (https://github.com/Azure/autorest/blob/master/Documentation/swagger-extensions.md#x-ms-long-running-operation)